ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -90,6 +108,27 @@ message CreatedEvent {
   ContractMetadata metadata = 10;
 }
 
+// View of a create event matched by an interface filter.
+message InterfaceView {
+
+  // The interface implemented by the matched event.
+  // Required
+  Identifier interface_id = 1;
+
+  // Whether the view was successfully computed, and if not,
+  // the reason for the error. The error is reported using the same rules
+  // for error codes and messages as the errors returned for API requests.
+  // Required
+  google.rpc.Status view_status = 2;
+
+  // The value of the interface's view method on this event.
+  // Set if it was requested in the ``InterfaceFilter`` and it could be
+  // sucessfully computed.
+  // Optional
```

Comment:
What language version are we using to serialize this value? I see two options:
1. The language version of `interface_id`
2. The language version of the data type that defines the interface view type.

I'm in favour of 1. because we can determine the interface_id's language version when we want to parse the view.

Reply:
Definitely 1 although I think it doesn’t actually matter at the moment. On the ledger API we don’t have an explicit version in the value and we don’t store those values.